src/js/generateLottos.js

```diff
@@ -1,9 +1,10 @@
 import { LOTTO_LENGTH, LOTTO_NUMBER_RANGE_MAX, LOTTO_NUMBER_RANGE_MIN } from './constants.js';
 
+const lottoNumbers = Array.from({ length: LOTTO_NUMBER_RANGE_MAX }).map(
+  (_, i) => i + LOTTO_NUMBER_RANGE_MIN
+);
```

Comment:
```suggestion
const lottoNumbers = Array(LOTTO_NUMBER_RANGE_MAX)
  .fill(LOTTO_NUMBER_RANGE_MIN)
  .map((min, index) => min + index);
```
이런 표현 방법도 있답니다!
사실 큰 차이는 없어요 ㅎㅎ
편한걸 써주세요!

Reply:
또 다른 방식을 알게되었습니다!
감사합니다 ㅎㅎ